docker-compose.yml

```diff
@@ -8,20 +8,41 @@ services:
       - POSTGRES_USER=${DB_USER}
 
   court_server:
-    build: ./packages/server
+    build: .
     image: aragon/court-server
+    command: npm run start:server:dev
     depends_on:
       - court_server_db
     ports:
-      - "8080:8080"
-      - "8081:8081"
+      - "8000:8000"
+    volumes:
+      - ./packages/server/src:/court-backend/packages/server/src
```

Comment:
Nice addition! I guess this is for "hot loading". I wonder if it would be useful to have `build` folder too.

Reply:
Yea! Actually `nodemon` should avoid building the package, it is set up to work with `babel-node` directly